### PR TITLE
first draft infrastructure episode

### LINKED ISSUE #6: Draft episode content for `lesson-infrastructure.md`
- [ ] reading list
    - a list of pages/publications trainees should read before joining this group discussion.
- [ ] discussion questions
    - questions to prompt interesting and relevant discussion around the themes of the week's reading.
    - it is totally fine - perhaps better, even - if these are questions that we do not have an answer to, e.g. the discussions could be useful for informing and improving the CLDT curriculum. It will probably make them more interesting too!
    - in this case, trainees should have spent time before the session initialising a new lesson repository and completing the steps described in the CLDT episode, so it might be a good idea to include some discussion questions based on that process.
- [ ] (optional) homework task(s)
    - anything you think it could be helpful for trainees to work through on their own between group discussions
    - we should certainly not aim to have homework every week, especially where the reading list for next week is already long.

### PR DESCRIPTION
**Pull request by tobyhodges**
Closes #6

### CODE REVIEW

File: episodes/lesson-infrastructure.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ By the end of this session, participants should be able to...
 
 :::::::::::::::::::::::::::::::::::::::::::::::::: checklist
 
-## Reading List
+## Reading List & Homework
 
 - [CLDT episode: The Carpentries Workbench](https://carpentries.github.io/lesson-development-training/07-infrastructure.html)
-- FIXME
+  - Please follow the steps in the episode, creating and configuring a new lesson repository, while you read through the content.
```
Comment (sstevens2) [inline episodes/lesson-infrastructure.md:30]: Maybe we should prompt them to note down any issues they had?  They hopefully are already doing that for lesson content but even an issue that doesn't change the curriculum might be good to discuss this week.

Comment (sstevens2) [reply episodes/lesson-infrastructure.md:30]: It is already in the discussion below but just in case they don't read ahead

**Review by sstevens2 — APPROVED** [review]
Looks good to me

File: episodes/lesson-infrastructure.md (hunk 1 above)
Comment (tobyhodges) [reply episodes/lesson-infrastructure.md:30]:
```suggestion
  - Please follow the steps in the episode, creating and configuring a new lesson repository, while you read through the content. 
  - Note down any problems you encounter along the way, no matter whether they were minor (i.e. slowed you down or confused you temporarily, but did not stop you from progressing) or major (i.e. you had to give up or ask for help).
```

Comment (sstevens2) [reply episodes/lesson-infrastructure.md]: That change looks great!

**Pull request merged**